### DISCUSSION TITLE
Apr 28 2021 ISG full team meeting

### DISCUSSION THREAD

**Discussion by mli1** [SafetyGraphics/safetyGraphics#534] [Meetings]
# Attendees
- @jwildfire 
- @xni7 
- @JimBuchanan 
- @anzianorj 
- @tbajn 
- @wal615 
- @PHayashi 
- @MayaGans 
- @wal615 
- @hendricksonb 
- @MelvinSMunsaka 
- @mli1 
- Siu-Chi Sun
- Scott Wang
- Gabrielle Murashova
- Nilesh Patel
- Danae Williams
- Michael Colopy
- Greg Ball
- Yu-Yi Hsu

# Agenda/Notes

1. Technical updates - Should be starting V2 alpha testing within the next few weeks! See the [v2 project board](https://github.com/SafetyGraphics/safetyGraphics/projects/14) for full details on V2 development. 
2. Walkthrough of Safety Explorer Suite: This is a JS html developed by Jeremy when he was at Rho. The webpage is as following--- https://rhoinc.github.io/safety-explorer-suite/test-page/queries/. The tool is quite mature and has a lot of useful charts there. Wanted to make the team be aware of its existence.  The tool was created mainly to deal with data from single study, and does not do study size adjusted rate yet.  
3. Call for volunteers if any of the existing chart in the Safety Explorer is super helpful, then we can consider developing some user guide. 
4. Reminder to the team members of the May 21 team building event.
5. All the subgroup updates will be moved to the May 19 team meeting due to time constraint.  Urgent items should be communicated to the team via email.

# Action Items

- No link to the Safety Explorer Suite on SafetyGraphics so far.  @jwildfire  will add the link and make sure the tools are all linked and are made aware to people. 
- Subgroup updates will be done on May 19 team meeting. ISG QT subgroup will meet CSRC on May 5, open to all the ISG members. (communicated as a post meeting action via email)

**Comment by jwildfire** [comment]
Thanks @mli1! I updated Agenda item #1 above.
